pkg/container-collection/containers.go

```diff
@@ -37,6 +37,9 @@ type Container struct {
 	// Pid is the process id of the container
 	Pid uint32 `json:"pid,omitempty" column:"pid,template:pid,hide"`
 
+	// CreatedAt represent the creation time of the container in time.RFC3339
+	CreatedAt string `json:"createdAt,omitempty"`
```

Comment:
Do we need this timestamp here? Or is it something we can calculate in `cmd/local-gadget/containers.go` when we receive the notification? 

IMO we should try to keep our representation of containers as small as possible and only add new members when totally required. 

@alban any opinions on this?

Reply:
> Do we need this timestamp here? Or is it something we can calculate in `cmd/local-gadget/containers.go` when we receive the notification?

We mainly need it for initial containers like to show users when the runtime created the container (it can be a week ago time) so tricky to calculate it but for new containers we can just show the current time. 

Having said above, I was wondering if we can skip showing timestamp or do you think it is nice to have it in `ContainerEvents`?

Reply:
We can print the current time in [timestamp](https://github.com/inspektor-gadget/inspektor-gadget/pull/1167/files#diff-4649a03815777bd0b520ade836043a3bb4baad34f2ec2e22f8d2b8277110082cR40). Should simplify all the changes to `containers-utils`.

Reply:
I agree with printing the current timestamp.